integration-cli/docker_cli_kill_test.go

```diff
@@ -40,6 +40,29 @@ func (s *DockerSuite) TestKillContainer(c *check.C) {
 
 }
 
+func (s *DockerSuite) TestKillofStoppedContainer(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "top")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal(out, err)
+	}
+
+	cleanedContainerID := strings.TrimSpace(out)
+
+	stopCmd := exec.Command(dockerBinary, "stop", cleanedContainerID)
+	if out, _, err = runCommandWithOutput(stopCmd); err != nil {
+		c.Fatalf("failed to stop container: %s, %v", out, err)
+	}
+
+	killCmd := exec.Command(dockerBinary, "kill", cleanedContainerID)
+	/* Container should be stopped, second kill should return an error */
+	if _, _, err = runCommandWithOutput(killCmd); err == nil {
```

Comment:
Can you check to make sure your text is displayed? Otherwise we can't be sure we're hitting your code - it could have failed for some other reason by mistake.